src/tests/Interop/COM/NETClients/ConsumeNETServer/Program.cs

```diff
@@ -29,6 +29,23 @@ static void Validate_Activation()
             Assert.IsTrue(Marshal.IsComObject(test));
         }
 
+        static void Validate_Activation_CreateInstance()
+        {
+            Console.WriteLine($"{nameof(Validate_Activation_CreateInstance)}...");
+
+            Type t = Type.GetTypeFromCLSID(Guid.Parse(Guids.ConsumeNETServerTesting));
+            Assert.IsTrue(t.IsCOMObject);
+
+            // Use the overload that takes constructor arguments. This tests the path where the runtime searches for the
+            // constructor to use (which has some special-casing for COM) instead of just always using the default.
+            object obj = Activator.CreateInstance(t, Array.Empty<object>());
```

Comment:
Should we also use the `CreateInstance` without the args overload?

Reply:
I just did this overload since all the COM + dynamic tests go through `Activator.CreateInstance(Type type)`. I can add it explicitly here too.